### DISCUSSION TITLE
Installing things in /usr/local/bin

### DISCUSSION THREAD

**Discussion by below** [Homebrew/discussions#3066] [Getting started]
Hello,

I have read around as much as I could, and I understand that on Apple Silicon, homebrew no longer installs itself or other software in `/usr/local`.
Now, especially Xcode is not reading the environment of the shell, and thus does not care what anyone puts into `.bash_profile`, `.zshrc`, or the like.

What is the "best practice", if I need to make something (say `node`) available to Xcode? The StackOverflow answer is to manually create a link in `/usr/bin/local`, which sounds too much of a hack for my taste.

I know the brew community is pretty good on these things, especially thinking about side-effects of various approaches. So, does anyone have a good idea?

**Comment by SMillerDev** [comment]
`/etc/paths.d/` is a good solution since it's system wide.
https://unix.stackexchange.com/questions/342574/use-etc-paths-or-etc-paths-d-to-add-items-to-the-path-in-macos-sierra 

If you don't want anything system-wide https://github.com/Homebrew/discussions/discussions/3035 will have some suggestion

**Reply by below** [reply]
Thank you! Would this be an idea for an improvement, so that `brew link` creates entries in that place? (And removes them on uninstall?)

**Reply by SMillerDev** [reply]
No, brew just manages it's own install. It's up for users to manage their PATH to optionally include brew

**Reply by below** [reply]
I have to unmark this as the answer, because Xcode.app (not the command line `xcodebuild`) apparently ignores `/etc/paths` and `/etc/paths.d`, just like all the other files.
